notebooks/advanced/Time_Series_Forecasting_with_Unstructured_Data_and_Amazon_SageMaker_Neural_Topic_Model/3_Forecast.ipynb

```diff
@@ -0,0 +1,972 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Amazon Forecast"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we will use Amazon Forecast, a fully managed service for producing time series forecasts using machine learning. \n",
+    "\n",
+    "We will call Forecast APIs using SageMaker for which we need to ensure that the SageMaker role associated with this Notebook environment has the AmazonForecastFullAccess policy attached to it. Please go to the IAM console and check to make sure that the role associated with the notebook has this policy attached.\n",
+    "\n",
+    "We also need to ensure that Forecast can access data in S3 buckets. To ensure this, in the IAM console, create a role called **Forecastbasicrole** which has AmazonS3FullAccess policy attached to it.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Libraries and load data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import sys\n",
+    "import os\n",
+    "import copy\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.metrics import mean_squared_error\n",
+    "import time\n",
+    "import datetime as dt\n",
+    "from time import sleep\n",
+    "import boto3\n",
+    "import sagemaker\n",
+    "# importing forecast notebook utility from notebooks/ directory\n",
+    "sys.path.insert( 0, os.path.abspath(\"./util\") )\n",
+    "import util"
+   ]
```

Comment:
Delete Delete "amazon-forecast-samples/notebooks/advanced/Time_Series_Forecasting_with_Unstructured_Data_and_Amazon_SageMaker_Neural_Topic_Model/util", and use "amazon-forecast-samples/notebooks/common/util" instead.